packages/bpk-component-info-banner/src/BpkInfoBannerInner.tsx

```diff
@@ -189,8 +178,11 @@ const BpkInfoBannerInner = ({
       {...rest}
     >
       <section className={sectionClassNames} role="presentation">
-        <div
-          role={isExpandable ? 'button' : undefined}
+        <BannerHeader
+          aria-label={isExpandable ? toggleButtonLabel : undefined}
```

Comment:
We shouldn't set the aria-label here, as it will override the visible text. (Look at the prev. comment)

Ideally the button screen reader should read:
`{visible text}, {toggleButtonLabel}, <expanded|collapsed>, button` 

We also need to review the 'aria-controls' for this expand/collapse behaviour:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-controls